breaking-front/src/pages/PostWrite/PostWrite.styles.js

```diff
@@ -0,0 +1,169 @@
+import Button from 'components/Button/Button';
+import styled, { css } from 'styled-components';
+
+export const PostWriteTitle = styled.h3`
+  font-size: 24px;
+`;
+
+export const PostRadioButton = styled(Button)`
+  ${({ id, radioContorl }) =>
+    id === radioContorl &&
+    css`
+      background-color: ${({ theme }) => theme.blue[400]};
+    `}
+`;
+
+export const PostSubmitButton = styled(Button)`
+  display: block;
+  width: 170px;
+  margin: 0 auto 50px;
+  background-color: ${({ theme }) => theme.blue[400]};
+`;
+
+export const Container = styled.div``;
```

Comment:
```suggestion
export const Container = styled.div`
  width: 800px;
  margin: 0 auto;
`;
```
이렇게 하는게 커보이지 않고 좋은 것 같습니다.